.eslintignore

```diff
@@ -2,3 +2,4 @@
 /dist/
 /coverage/
 /lib/
+/scripts/
```

Comment:
We definitely want linting in here, since this is code we write ourselves. Yes, these are not part of the final build, but the code should still be of high quality.

Reply:
I had ignored the whole folder because it wasn't in the build but that's a good point. Only ignoring the tmp folder now